### PR TITLE
Add hmpps-book-secure-move-api-dev environment

### PR DESCRIPTION
**Pull request by martyn-w**
This adds a new environment, `dev` for the hmpps-book-secure-move-api project, to facilitate dev testing outside of staging / pre-production.

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/rds.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+module "rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.0"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/rds.tf]: The RDS module is now on release 5.1 Please use that and make any changes necessary to your code.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/s3.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
+ Based on https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/tree/master/example
+ */
+module "book_a_secure_move_documents_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.0"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/s3.tf]: The S3 module is now on release 4.1 - please use that version

**Comment by martyn-w** [comment]
Hi @digitalronin would you be able to merge this in (I don't have write access). Thanks.

**Comment by digitalronin** [comment]
@martyn-w Aren't you a member of the MoJ github organisation?

**Comment by martyn-w** [comment]
@digitalronin yes I am - but I don't have write access to this repository:
Screenshot 2020-02-18 at 16 00 10

**Comment by digitalronin** [comment]
Ah, you should have. I've just added write access for the `book-a-secure-move` team.

I'll merge this one, but you should be able to just branch and push, from now on.

**Comment by martyn-w** [comment]
Thanks @digitalronin 👍

**Pull request merged**